pkg/client/factory.go

```diff
@@ -29,6 +38,7 @@ func NewFactory(baseName string) Factory {
 	}
 
 	f.flags.StringVar(&f.kubeconfig, "kubeconfig", "", "Path to the kubeconfig file to use to talk to the Kubernetes apiserver. If unset, try the environment variable KUBECONFIG, as well as in-cluster configuration")
+	f.flags.Uint16Var(&f.httpPort, "httpPort", uint16(defaultHttpPort), fmt.Sprintf("Port to serve charts on.  Default %v", defaultHttpPort))
```

Comment:
```$ ./kubechart --help
Monitor Pod phase transitions over time in a Kubernetes cluster.

Usage:
  kubechart [flags]

Flags:
      --alsologtostderr                  log to standard error as well as files
  -h, --help                             help for kubechart
      --httpPort uint16                  Port to serve charts on.  Default 3000 (default 3000)
```
the command library adds the default automatically so you can remove it.

also, let's call this flag `http-port` (to match kube-style flags) and the argument `port` rather than `uint16`.

Reply:
Not quite sure how to make the argument come out as "port" rather than "uint16".  I can define a port type as uint16, but then I can't use a `Uint16Var` flag with it.

Reply:
ha! nevermind. didn't realize that was generated.